daml/FOO-wallet-amulet-proxy/README.md

```diff
@@ -0,0 +1,7 @@
+# Example
```

Comment:
@moritzkiefer-da : could you have a peek before I share this more widely with wallet providers? thx a lot.